BTrees/_base.py

```diff
@@ -1269,7 +1269,7 @@ def iteritems(self, min=_marker, max=_marker,
 
     def byValue(self, min):
         return reversed(
-                sorted((v, k) for (k, v) in self.iteritems() if v >= min))
+                sorted(((v, k) for (k, v) in self.iteritems() if v >= min)))
```

Comment:
I thought we needed ``cmp``, but this is sorting *values* not keys. When I changed it back, I left the extra parents.

I'll revert this.